### PR TITLE
[Packaging][Wheel] Make nightly wheel links available on crossbow's static github page

### PR DESCRIPTION
**Pull request by kszucs**
`python crossbow.py github-page update-wheel-links -n 30` command generates a static github page like: http://kszucs.me/crossbow/nightly/wheel/

This enables to install nightly wheels via pip: `pip install --no-index --find-links http://kszucs.me/crossbow/nightly/wheel/2019-04-24/ pyarrow`

The urls will change of course.

### CODE REVIEW

File: dev/tasks/crossbow.py [hunk 1]
```diff
@@ -534,6 +578,92 @@ def crossbow(ctx, github_token, arrow_path, queue_path):
...
+              help='Just render the files without pushing')
+@click.pass_context
+def update_wheel_links(ctx, n, gh_branch, job_prefix, dry_run):
+    def generate_content(links):
+        links = ['<li><a href="{}">{}</a></li>'.format(url, name)
+                 for name, url in sorted(links.items())]
+        return '<html><body><ul>{}</ul></body></html>'.format(''.join(links))
+
+    queue = ctx.obj['queue']
+    # queue.fetch()
```
Comment (kszucs) [inline dev/tasks/crossbow.py]: Uncomment it

**Comment by xhochy** [comment]
Do we already have a place where will upload this?

**Comment by wesm** [comment]
I believe we need a simple nightly status page with links to Crossbow builds rather urgently to increase transparency of failing Crossbow builds where there currently is ~none (unless you know where to look -- ideally the Arrow mailing list would be notified about failing or flaky builds). This should be the most quick-and-dirty thing possible to have a list indicating whether things passed or failed. Buildbot etc. does not need to be involved. It seems like this patch does part of what's needed

cc @nealrichardson

**Comment by wesm** [comment]
Should probably discuss this on the mailing list

**Comment by emkornfield** [comment]
@wesm is it reasonable to separate this change from the discussion on how to get status of cross-bow builds?

**Comment by wesm** [comment]
Yes, I'm closing this until we come up with a more holistic and comprehensive plan about organizing the information that is produced by Crossbow tasks. We are just finally about to have e-mail notifications about failures, but there is a lot more we should do

**Pull request closed without merging**